python/modules/IcePy/BatchRequestInterceptor.cpp

```diff
@@ -133,55 +133,22 @@ static PyMethodDef BatchRequestMethods[] = {
      METH_NOARGS,
      PyDoc_STR("getProxy() -> Ice.ObjectPrx")},
     {"enqueue", reinterpret_cast<PyCFunction>(batchRequestEnqueue), METH_NOARGS, PyDoc_STR("enqueue() -> None")},
-    {0, 0} /* sentinel */
+    {nullptr, nullptr} /* sentinel */
 };
 
 namespace IcePy
 {
+    // clang-format off
```

Comment:
I disable clang-format for PyTypeObject, to avoid extraneous indentation after the `PyVarObject_HEAD_INIT` macro